applications/ParticleMechanicsApplication/custom_python/particle_mechanics_python_application.cpp

```diff
@@ -156,6 +156,7 @@ PYBIND11_MODULE(KratosParticleMechanicsApplication, m)
     KRATOS_REGISTER_IN_PYTHON_3D_VARIABLE_WITH_COMPONENTS(m,  AUX_R_ACC )
     KRATOS_REGISTER_IN_PYTHON_3D_VARIABLE_WITH_COMPONENTS(m,  AUX_T_ACC )
     KRATOS_REGISTER_IN_PYTHON_VARIABLE(m,  NODAL_LUMPED_MASS )
+    KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, IS_STRUCTURE)
```

Comment:
Use the IS_STRUCTURE deprecated variable (deprecated_variables.h) in the core instead of adding a new one in your application.

Reply:
In addition it is dangerous to declare sth that is probably going to be used as "bool" (I am assuming here) as a double

Reply:
@philbucher you are completely right, but this is so for legacy reasons (see #214). Hopefully we can fix it soon, but not on this PR.

:+1:  to @rubenzorrilla's comment: please use the variable in the core.

Reply:
Thanks. Will change to the depcrecated_variables in the core.